app/assets/javascripts/discourse/app/controllers/preferences/interface.js

```diff
@@ -116,6 +130,30 @@ export default Controller.extend({
     }
   },
 
+  @discourseComputed
+  userSelectableColorSchemes() {
+    return listColorSchemes(this.site);
+  },
+
+  @discourseComputed("userSelectableColorSchemes")
```

Comment:
pretty sure you can one line it:

showColorSchemeSelector: reads("userSelectableColorSchemes.length")